interface/forms/questionnaire_assessments/table.sql

```diff
@@ -0,0 +1,22 @@
+--
```

Comment:
wouldn't even include this file since this is installed by default (then avoid needing to remember to update tables in 2 places when make changes in schema).

Reply:
True. I pondered to leave or not but your reason is a good one. will remove.